bundle/python_bundle_workflow/scripts/train.py

```diff
@@ -110,27 +110,23 @@ def _get_property(self, name, property):
             self._props[name] = value
         else:
             try:
-                value = self.__getattribute__(f"_{name}")
+                value = getattr(self, f"get_{name}")()
             except AttributeError:
                 if property[BundleProperty.REQUIRED]:
                     raise ValueError(f"unsupported property '{name}' is required in the bundle properties.")
```

Comment:
perhaps this error message can provide more info about how to resolve the error? like adding the property? create `get_{name}` method?

Reply:
Sure, added in the latest commit.

Thanks.